Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -53,28 +53,114 @@
+      maxVol: 10
+      reagents:
+      - ReagentId: Dermaline
+        Quantity: 10
+
+- type: entity
+  name: ephedrine autoinjector cartridge
+  parent: BaseCartridgeAutoinjector
+  description: Contains 10u of ephedrine, used in a cartridge autoinjector.
+  id: CartridgeEphedrine
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)